beets/importer.py

```diff
@@ -530,6 +533,17 @@ def remove_duplicates(self, lib):
                 util.prune_dirs(os.path.dirname(item.path),
                                 lib.directory)
 
+    def set_fields(self):
+        set_fields_dict = config['import']['set_fields']
+        for field in set_fields_dict.keys():
+            value = set_fields_dict[field].get()
+            log.debug(u'Set field {1}={2} for {0}',
+                      displayable_path(self.paths),
+                      field,
+                      value)
+            self.album[field] = value
+            self.album.store()
```

Comment:
I recommend *not* including a `store` call here. Instead, we can just modify the values before they ever hit the database at all! Another comment below will elaborate…